src/custom-element.js

```diff
@@ -648,15 +679,17 @@ function createBaseAmpElementProto(win) {
    * @final @this {!Element}
    */
   ElementProto.attachedCallback = function() {
-    if (!TEMPLATE_TAG_SUPPORTED) {
+    if (!TEMPLATE_TAG_SUPPORTED && this.isInTemplate_ === undefined) {
       this.isInTemplate_ = !!dom.closestByTag(this, 'template');
```

Comment:
When `<template>` is supported, the elements are not activated, custom elements are not instrumented and none of these callbacks are called. Correctly, the nodes inside `template` are treated as if belonging to a separate document and only instrumented when imported into the main document (`importNode`). But browsers that do not support `<template>` (IE) the template nodes are instrumented as normal nodes. 

An easy test is this:
```
<template>
  <style>
    .body {background: red;}
  </style>
</template>
```

When `template` is supported, there are no sideeffects. When not supported - the whole body will be painted red.